src/ElementQueries.js

```diff
@@ -12,7 +12,7 @@
     } else {
         root.ElementQueries = factory(root.ResizeSensor);
     }
-}(this, function (ResizeSensor) {
+}(typeof window !== 'undefined' ? window : this, function (ResizeSensor) {
```

Comment:
The solution to this specific big report looks fine to me, but I'm curious as to why it's not working the way it's written. The implementation of root was taken right from the UMD example code, so I'd expect this to be a problem with any module using the UMD module pattern.

Any insights in your bugfinding @newlukai?